### PR TITLE
Fix Issue #11: `gem crashes if no default config file present`

### PR DESCRIPTION
**Pull request by yverbytskyi**

### CODE REVIEW

File: lib/passpartu.rb [hunk 1]
```diff
@@ -27,23 +30,44 @@ def self.configure
...
+
+    def wrong_config_error
+      <<-ERROR
+
+        ************************************************************************************
+        !!! Passpartu is not configured properly. Check configuration file: #{@policy_file}
+        :: Configuration example: https://github.com/coaxsoft/passpartu#real-life-example ::
+        ************************************************************************************
+      ERROR
+    end
```
Comment (OrestF) [inline lib/passpartu.rb]: Move two methods above into `initializer.rb`

File: lib/passpartu.rb [hunk 2]
```diff
@@ -27,23 +30,44 @@ def self.configure
...
   end
 
   configure {}
 end
 
-initializer = './config/initializers/passpartu.rb'
-require initializer if File.exist?(initializer)
+require Passpartu::Initializer::DEFAULT_INITIALIZER_PATH
```
Comment (OrestF) [inline lib/passpartu.rb]: Could u require `DEFAULT_INITIALIZER_PATH ` without `Passpartu::Initializer`?

Comment (yverbytskyi) [reply lib/passpartu.rb]: it will cause unitialized constant error here

File: passpartu.gemspec [hunk 3]
```diff
@@ -29,14 +32,10 @@ Gem::Specification.new do |spec|
...
   # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  end
+  spec.files = `git ls-files config lib LICENSE.txt README.md`.split($INPUT_RECORD_SEPARATOR)
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.files = Dir['README.md', 'lib/**/*', 'lib/*', 'passpartu.gemspec']
```
Comment (yverbytskyi) [inline passpartu.gemspec]: Double assignment of `spec.files` was present.

**Pull request closed without merging**